source/includes/v2/_captions.md

```diff
@@ -0,0 +1,319 @@
+# Caption API
+
+The Caption API allows you to request and manage closed captioning on your Videos.
+
+## Caption Resource
+
+The Caption Resource(s) returned in a successful response.
+
+> Example Caption Resource Object.
+
+```json
+{
+  "identifier": "...",
+  "file_url": "...",
+  "language": "...",
+  "status": "...",
+  "type": "...",
+}
+```
+
+| Prop | Type | Value |
+| ---- | ---- | ----- |
+| **identifier** | string | The unique identifier for the caption on VidGrid. |
+| **file_url** | string | URL to the VTT file of a `COMPLETED` caption resource. |
+| **language** | string | The language of the caption file. |
+| **status** | string | The current status of this caption resource.<br>*Possible values: `QUEUED`,`PENDING`,`IN_PROGRESS`,`CANCELED`,`FAILED`,`COMPLETED`*|
+| **type** | string | The type of caption resource.<br>*Possible values: `machine`,`professional`,`upload`*|
+
+## Create Caption
+
+This endpoint creates a caption request for a [Video Resource](#video-resource) and returns a [Caption Resource](#caption-resource).
```

Comment:
I think it'd be more accurate to say that it returns something that describes the caption request has been created instead of the actual caption resource. chances are it won't be done at the end of the request

Reply:
good call